yarn-project/end-to-end/src/spartan/upgrade_rollup_version.test.ts

```diff
@@ -129,112 +115,6 @@ describe('spartan_upgrade_rollup_version', () => {
         defaultL1TxUtilsConfig,
       );
 
-      await updateSequencersConfig(config, {
-        governanceProposerPayload: payloadAddress,
-      });
-
-      const rollup = new RollupContract(l1PublicClient, originalL1ContractAddresses.rollupAddress.toString());
-      const governanceProposer = new GovernanceProposerContract(
-        l1PublicClient,
-        originalL1ContractAddresses.governanceProposerAddress.toString(),
```

Comment:
again, im pretty scared of explicitly removing all of this

Reply:
Adding an explicit solidity test to showcase the flow. Also, we have an E2E for doing the votes and such so it is essentially already there in the case without killing nodes to see connection so should be similarly tested but with reduced runtime